src/it/setup-it-parent/pom.xml

```diff
@@ -41,7 +41,8 @@
 
         <mavenVersion>3.3.9</mavenVersion>
 
-        <protobufVersion>3.6.1</protobufVersion>
+        <protobufVersion>3.9.1</protobufVersion>
```

Comment:
This version was also used in CI configuration and the docs.
While we are at it, let's update to the current latest version of protobuf/protoc (`3.11.2` at the time of writing) and update all other places where `3.6.1` was used.

Basically, do a full text search for `3.6.1` and replace all occurrences with `3.11.2`.

Reply:
Updated to 3.11.1 since 3.11.2 is not yet on Maven central. Anyway there are no changes for Java since 3.11.0 according to the release notes.